### PR TITLE
orm, blog auth

### PR DESCRIPTION
**Pull request by ybrodsky**

### CODE REVIEW

File: es/tutorials-and-examples/blog-auth-example/auth.rst [hunk 1]
```diff
@@ -1,15 +1,357 @@
...
+
+
+Autorización (quién está autorizado a acceder qué)
+==================================================
+
+Como mencionamos antes, estamos convirtiendo este blog en una herramienta de autoría multiusuario, y para hacer esto necesitamos modificar la tabla de posts para agregar referencia al modelo User::
+
+    ALTER TABLE articles ADD COLUMN user_id INT(11);
+
+También, un pequeño cambio en ArticlesController es necesario para guardar el usuario logeado como referencia en los artículos creados:
```
Comment (markstory) [inline es/tutorials-and-examples/blog-auth-example/auth.rst]: This line needs to end in `::` so the next block is pre-formatted.

**Comment by ybrodsky** [comment]
That's the only mistake? 'Cause travis-ci is marking a mistake in line 229, unexpected indent, among others.

**Comment by markstory** [comment]
@ybrodsky If the build tooling is finding issues in your formatting, it is probably correct :smile:

File: es/orm.rst [hunk 2]
```diff
@@ -1,26 +1,108 @@
...
+
+El ORM incluído en CakePHP se especializa en base de datos relacionales, pero puede ser extendido 
+para soportar alternativas.
+
+El ORM de CakePHP toma ideas y conceptos de los modelos ActiveRecord y Datamapper. Aspira a 
+crear una implementación híbrida que combine aspectos de los dos modelos para crear un ORM rápido 
+y fácil de usar.
+
+Antes de comentar explorando el ORM, asegurate de configurar tu conexion :ref:`configure your
+database connections <database-configuration>`.
```
Comment (cake17) [inline es/orm.rst]: It refers to a label database-configuration that does not yet exists in es files.
Please just add the following paragraph in [this file](https://github.com/cakephp/docs/blob/3.0/es/orm/database-basics.rst) after the ``..note``
```
.. _database-configuration:

Configuration
=============
```
This should stop the sphinx warning

**Comment by ybrodsky** [comment]
Well, there's a new one. =/

**Comment by cake17** [comment]
@ybrodsky not your fault, it comes from a previous commit. I'll fix this post-merge. Thanks for your great work ! :+1:

**Comment by ybrodsky** [comment]
Thanks for the help.

**Pull request merged**